dashboard/test/models/user_test.rb

```diff
@@ -1393,4 +1393,38 @@ def create_user_with_username(username)
     create(:plc_user_course_enrollment, plc_course: (create :plc_course), user: user)
     assert user.should_see_inline_answer?((create :script_level))
   end
+
+  test 'do not show race interstitial to teacher' do
+    user = create :teacher
```

Comment:
nit: My vote would be to name teacher users as teacher, and student users as student, i.e., `teacher = create :teacher` and `student = create :student`.